classes/Admin/Shortcode/UI.php

```diff
@@ -149,7 +149,7 @@ public static function shortcode_ui_var() {
 				'sections'       => $shortcode->_subtabs(),
 				'fields'         => $shortcode->_fields(),
 				'has_content'    => $shortcode->has_content,
-				'ajax_rendering' => $shortcode->ajax_rendering === true,
+				'ajax_rendering' => $shortcode->ajax_rendering === true,  // phpcs:ignore
```

Comment:
@s-acv2 For these, and future reference ask in slack if your not sure, the goal is to never ignore if possible.

This one is a boolean already, so could probably just be shortened to
```
'ajax_rendering' => $shortcode->ajax_rendering
```